src/components/Button/Button.stories.tsx

```diff
@@ -4,10 +4,10 @@ import { ComponentStory, ComponentMeta } from "@storybook/react";
 import Button from "./Button";
 
 export default {
-  title: "Example/Button",
+  title: "Button",
```

Comment:
Actually, in general for this, I think lebih bagus kalo lu kasi variant dia dimana dia disabled, biar orang langsung tau since disabled itu mayan penting